packages/react-charting/docs/TestPlans/DonutChart/ComponentTests.md

```diff
@@ -0,0 +1,21 @@
+<div align="center">
+
+# Component testing - Donut chart test plan
+
+![Alt text](BasicDonutChart.png)
+
+### Subcomponents: Pie and Legend
+
+### Library used: jest and (enzyme or react testing library)
+
+</div>
```

Comment:
I am seeing table not aligning properly. Can you check.

Reply:
resolved